docs/src/main/sphinx/connector/snowflake.md

```diff
@@ -92,3 +92,24 @@ statements, the connector supports the following features:
 - {doc}`/sql/alter-table`
 - {doc}`/sql/create-schema`
 - {doc}`/sql/drop-schema`
+
+## Performance
+
+The connector includes a number of performance improvements, detailed in the
+following sections.
+
+(snowflake-pushdown)=
+### Pushdown
+
+The connector supports pushdown for a number of operations:
+
+{ref}`Aggregate pushdown <aggregation-pushdown>` for the following functions:
+
+- {func}`avg` for bigint, floating point and decimal types
+- {func}`count` for bigint types
```

Comment:
Other connectors don't mention the supported types. Do you plan to update other docs as well?

Reply:
maybe I will leave it out here .. too much to dig into at the moment to get that more detailed coverage